solidity/scripts/pricing-test.js

```diff
@@ -44,6 +44,42 @@ module.exports = async function() {
                 prevRewards[i] = (await availableRewards(accounts[i], contractOperator, contractStatistics)).toString();
             }
 
+
+            let rewardsSum = web3.utils.toBN(0);
+            for (let i = 0; i < accounts.length; i++) {
+                rewardsSum = rewardsSum.add(web3.utils.toBN(prevRewards[i]));
+            }
+
+            const serviceContractBalance = await web3.eth.getBalance(contractService.address);
+            const dkgFeePool = await contractService.dkgFeePool();
+            const requestSubsidyFeePool = await contractService.requestSubsidyFeePool();
+            const dkgContributionMargin = await contractService.dkgContributionMargin();
```

Comment:
Those functions are not currently exposed on the service contract. I'll add them in a separate PR. Let's hold on with merging this one.

Reply:
Done in https://github.com/keep-network/keep-core/pull/1738